gateway/handle_http_auth_test.go

```diff
@@ -76,6 +77,21 @@ func TestAuth(t *testing.T) {
 			body, err := io.ReadAll(w.Body)
 			require.NoError(t, err, "reading response body should succeed")
 			require.Equal(t, "Failed to read writeKey from header\n", string(body))
+			require.Equal(t,
```

Comment:
Could probably use `require.EqualValues` if you want to avoid the cast below (i.e. `float64(1)`).